resources/lib/vrtplayer/vrtplayer.py

```diff
@@ -87,14 +87,28 @@ def show_category_menu_items(self):
         self._kodi_wrapper.show_listing(category_items, sort='label', content_type='files')
 
     def show_livestream_items(self):
+
+        if self._kodi_wrapper.has_white_icons():
```

Comment:
people will only need the if when they manually uninstall the white icons addon. I find it a bit unnecessary

Reply:
I added it as a dependency to the addon.xml so the conditional is no longer necessary. Unless you do not want to have it as a dependency (just optional).

Reply:
If you accept the new dependency, I can indeed remove the conditional statements.

PS If we have the coloured icons as well, I'd like to use the white icons for the fanart, and coloured for the thumbnail/icon.

Reply:
To me it looks good feel free to merge already :)